WasatchNET/MockSpectrometer.cs

```diff
@@ -308,7 +308,8 @@ public override void close()
         }
 
         public override bool isARM => false;
-
+        public override bool isInGaAs => InGaAs;
+        bool InGaAs = false;

```

Comment:
I don't understand why you need both isInGaAs and InGaAs?

Reply:
it's a mock spectrometer overriding an in-lined function that returns a value based on USB characteristics. It has to pull it from somewhere and I figured we probably would want to have the option to make a mock InGaAs

Reply:
I get that, I just don't understand why the mock InGaAs couldn't literally set the existing isInGaAs to true?

Reply:
I don't believe it's possible unless I also change isInGaAs in base spectrometer, because it's a property with no underlying structure. There would be nowhere for the mock to store it because there is no underlying field, it's just a pass through with no setter. I am hesitant to change it in base because the analogous fields isARM and isSIG are also just pass throughs